packages/animations/example/lib/main.dart

```diff
@@ -74,7 +74,7 @@ class _TransitionsHomePageState extends State<_TransitionsHomePage> {
                   title: 'Fade through',
                   subtitle: 'FadeThroughTransition',
                   onTap: () {
-                    Navigator.of(context).push(
+                    Navigator.of(context)!.push(
```

Comment:
Is there a push to make Navigator.of(context) nnbd similar to Theme.of(context). ?

Reply:
Navigator.of(context) is defined to return a `NavigatorState?` inside flutter/flutter, so we can't do anything about it.